test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -312,8 +312,11 @@ private class DataPoint
             public float[] Features { get; set; }
         }
 
-        [Fact]
-        void LpNormOnnxConversionTest()
+        [Theory]
+        [CombinatorialData]
+        public void LpNormOnnxConversionTest(
+            bool ensureZeroMean,
+            LpNormNormalizingEstimatorBase.NormFunction norm)
```

Comment:
I assume it will infer all possible values in this case since they are discrete?